app_demo.rb

```diff
@@ -0,0 +1,59 @@
+require './lib/order'
+class Takeaway
+
+  attr_accessor :menu1, :order1, :name, :price, :menu_item1
+
+  def initialize
+    @menu1 = Menu.new Dish
+  end
+
+
+  def demo
```

Comment:
Metrics/CyclomaticComplexity: Cyclomatic complexity for demo is too high. [10/6]<br>Metrics/AbcSize: Assignment Branch Condition size for demo is too high. [50.07/15] (http://c2.com/cgi/wiki?AbcMetric)<br>Metrics/MethodLength: Method has too many lines. [43/10] (https://github.com/bbatsov/ruby-style-guide#short-methods)